### PR TITLE
chore: Added social icon LinkedIn, Issue #40

### PR DESCRIPTION
**Pull request by srideviraju**
### Check List

- [X] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [X] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [X] I have run `npx turbo format` to ensure the code follows the style guide.
- [X] I have run `npx turbo test` to check if all tests are passing.
- [X] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: components/__design__/social-logos.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import Image from 'next/image';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const SocialLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+    <div>
```
Comment (HinataKah0) [inline components/__design__/social-logos.stories.tsx]: Nit: Please help to fix the indentation so it looks like this:
```js
component: () => {
  return (
    <div>
    ...
    </div>
  );
},
```

And don't forget to fix the indentation in `<Image />` below.

**Review by HinataKah0 — COMMENTED** [review]
Left 1 minor comment.
Thank you for the contribution! 🎉

File: components/__design__/social-logos.stories.tsx [hunk 2]
```diff
@@ -16,6 +16,14 @@ export default {
...
           />
         </div>
+        <div>
+          <Image
+            src="/static/images/logos/social-linkedin.svg"
+            alt="LinkedIn Logo"
+            width={64}
+            height={64}
+          />
+        </div>
```
Comment (bmuenzenmeyer) [inline components/__design__/social-logos.stories.tsx]:
```suggestion
        </div>
```

**Review by bmuenzenmeyer — COMMENTED** [review]
trying to use the GUI to fix this prettier issue

**Review by bmuenzenmeyer — APPROVED** [review]

**Comment by bmuenzenmeyer** [comment]
thanks for your contribution to the project!

**Pull request merged**